HPXMLtoOpenStudio/measure.rb

```diff
@@ -2516,6 +2527,15 @@ def self.add_component_loads_output(runner, model, living_zone, liv_load_sensors
     mechvent_sensors.each do |sensor|
       program.addLine("Set hr_mechvent = hr_mechvent - #{sensor.name}")
     end
+    # Adjust component load reporting for preconditioning
```

Comment:
Adjust component loads read from equipment objects (add preconditioning loads to mechvent subitem).